src/søknad/forside/Forsideinformasjon.tsx

```diff
@@ -81,6 +83,7 @@ const Forsideinformasjon: React.FC<InnholdProps> = ({
           <Språkvelger />
         </FeltGruppe>
       )}
+      {locale == "en" && <AlertStripeAdvarsel>We are in the process of translating this application. The few missing translations will show up in Norwegian until we've translated them.</AlertStripeAdvarsel>}
```

Comment:
Er `will show up in Norwegian` riktig engelsk?

Tenker kanskje ´will be shown in Norwegian´ eller ´appear in Norwegian´ er bedre 🤔

Reply:
Tror det er riktig, men `appear in` er kanskje bedre. Endret i 4e21695